lib/mcb.rb

```diff
@@ -247,6 +247,9 @@ def iterate_v1_endpoint(url:, endpoint:, **opts)
     def remote_connect_options
       envs = env_to_azure_map.keys.join(', ')
       Proc.new do
+        option :R, 'recruitment_year',
+               "Set the recruitment year, defaults to 2019",
```

Comment:
I think the default is the current recruitment cycle, which happens to be 2019 now but will change in the future. Might be better to use that wording.